reference/functions/cf_version_minimum.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: cf_version_minimum
+published: true
+tags: [reference, utility functions, functions]
+---
+
+[%CFEngine_function_prototype(version_string)%]
+
+**Description:** 
```

Comment:
All of the other markdown files have content here in the Description. Why not this one?

Reply:
Mistake. : (